src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -977,7 +977,8 @@ public void StartInfo_NotepadWithContent(bool useShellExecute)
                 }
                 finally
                 {
-                    process.Kill();
+                    if (process != null && !process.HasExited)
+                        process.Kill();
```

Comment:
Right, I didn't protect that as I figure the test has failed. Lots of tests, if they fail, will throw NRE surely? Happy to improve the message, though.